spec/features/accessibility_spec.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Accessibility tests", driver: :selenium_chrome_headless do
+  it "audits the home page" do
+    visit root_path
+    expect(page).to be_accessible
+  end
```

Comment:
We would obviously want to add a whole lot more URLs here! This is a start.

Reply:
Yeah, we'd need to add the first-time welcome page. We presumably also want to add lots more pages (deposit form, collection pages, etc.).